api/src/api/opportunities_v1/opportunity_schemas.py

```diff
@@ -308,6 +308,60 @@ class OpportunityV1Schema(Schema):
     updated_at = fields.DateTime(dump_only=True)
 
 
+class OpportunityAttachmentV1Schema(Schema):
+    file_location = fields.String(
+        metadata={
+            "description": "The URL to download the attachment",
+            "example": "https://...",
+        }
+    )
+    mime_type = fields.String(
+        metadata={"description": "The MIME type of the attachment", "example": "application/pdf"}
+    )
+    file_name = fields.String(
+        metadata={"description": "The name of the attachment file", "example": "my_NOFO.pdf"}
+    )
+    file_description = fields.String(
+        metadata={
+            "description": "A description of the attachment",
+            "example": "The full announcement NOFO",
+        }
+    )
+    file_size_bytes = fields.Integer(
+        metadata={"description": "The size of the attachment in bytes", "example": 10012}
+    )
+    opportunity_attachment_type = fields.String(
+        metadata={
+            "description": "The type of attachment",
+            "example": "notice_of_funding_opportunity",
+        }
+    )
```

Comment:
We can make this an enum and point to the one you created in the prior PR to setup the DB table

Reply:
Updated this, thanks.